src/H5Dselect.c

```diff
@@ -401,7 +401,7 @@ H5D_select_io_mem(void *dst_buf, H5S_t *dst_space, const void *src_buf, H5S_t *s
 
             /* Perform vectorized memcpy from src_buf to dst_buf */
             if ((bytes_copied = H5VM_memcpyvv(dst_buf, dst_nseq, &curr_dst_seq, dst_len, dst_off, src_buf,
-                                              src_nseq, &curr_src_seq, src_len, src_off)) < 0)
+                                              src_nseq, &curr_src_seq, src_len, src_off, 0)) < 0)
```

Comment:
I'd maybe consider `SIZE_MAX` instead of 0 for an unknown value, but in this case we should probably make sure we always have a known value to pass to `H5VM_memcpyvv`.